File: coordinate_geometry.cpp
#include <cmath>
#include <iostream>
#include <math.h>
using namespace std;

int main() {
double x1, y1, x2, y2;
    cout << "Enter the x coordinate of point 1:";
    cin >> x1;
    cout << "Enter the y coordinate of point 1:";
    cin >> y1;
    cout << "Enter the x coordinate of point 2:";
    cin >> x2;
    cout << "Enter the y coordinate of point 2:";
    cin >> y2;

    double distance = sqrt(pow(x2 - x1, 2) + pow(y2 - y1, 2));

    cout << "The distance between points is " << distance << endl;
    return 0;
}

File: coordinate_geometry.py
import math

x1=float(input("Enter first point's x coordinate:"));
y1=float(input("Enter first point's y coordinate:"));
x2=float(input("Enter second point's x coordinate:"));
y2=float(input("Enter second point's y coordinate:"));

mesafe = math.sqrt((x2 - x1)**2 + (y2 - y1)**2);
print(f"Distance between these points is: {mesafe}");
